appium/tests/specs/live/login/LoginToMultipleAccounts.spec.ts

```diff
@@ -1,94 +1,93 @@
-// Temporary disabled because of https://github.com/FlowCrypt/flowcrypt-ios/issues/1383
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  SearchScreen,
+  EmailScreen,
+  MenuBarScreen
+} from '../../../screenobjects/all-screens';
+import { CommonData } from "../../../data";
+import BaseScreen from "../../../screenobjects/base.screen";
 
-// import {
-//     SplashScreen,
-//     SetupKeyScreen,
-//     MailFolderScreen,
-//     SearchScreen,
-//     EmailScreen,
-//     MenuBarScreen
-// } from '../../../screenobjects/all-screens';
-// import {CommonData} from "../../../data";
-// import BaseScreen from "../../../screenobjects/base.screen";
 
+describe('LOGIN: ', () => {
 
-// describe('LOGIN: ', () => {
+  // Temporary disabled because of https://github.com/FlowCrypt/flowcrypt-ios/issues/1383
+  xit('user should be able login to multiple accounts', async () => {
```

Comment:
As I checked test results were not reported not because of our new sempaphoreci structure.
It was not reported because of blank test file. This test file generated 0 byte test-result xml and therefore EOF error occurred.
So I used `xit` instead of commenting whole file.

![image](https://user-images.githubusercontent.com/77344191/174577017-19013dc4-6c29-439e-9a31-0e4ead0fc4d4.png)

Reply:
@sosnovsky

Reply:
Great finding 👍